drops/helpers/Interface.py

```diff
@@ -0,0 +1,35 @@
+import json
+import logging
+from http.client import HTTPConnection
+from multiprocessing import Queue, Semaphore
+from drops.helpers.ServerResponse import ServerResponse
+
+logger = logging.getLogger(__name__)
+
+'''
+send transmits a formatted HTTP GET request
+it will not check the validity of request
+it will persist result in a place that can be read.
+'''
+def send(conn : HTTPConnection, queue : Queue, q_ready : Semaphore, endpoint : str):
```

Comment:
Love where this is headed. Two pedantic notes:
1. Interface is too generic, lets call this HTTPInterface if anything. HTTPTranciever is somewhat more accurate and representative of how we're using the HTTP interface as the comms "bus" our middle is wrapping. 
2. I would personally make this a class. It is a bit tomato tomato BUT then the class can be instantiated with critical member variables: HTTPConnection, queue, queue_ready. `send` will only take on argument as well then.